src/java/org/apache/cassandra/db/repair/CassandraValidationIterator.java

```diff
@@ -195,12 +195,19 @@ public CassandraValidationIterator(ColumnFamilyStore cfs, Collection<Range<Token
             if (!isIncremental)
             {
                 // flush first so everyone is validating data that is as similar as possible
-                StorageService.instance.forceKeyspaceFlush(cfs.keyspace.getName(), cfs.name);
+                cfs.forceBlockingFlush(ColumnFamilyStore.FlushReason.REPAIR);
```

Comment:
It's hard for me to know whether it would actually be useful, but we could possibly make `REPAIR` more granular (ex. `VALIDATION`, `ANTICOMPACTION`, etc.)

Reply:
Might be helpful in the logs; done.